client/src/mergeUtils.js

```diff
@@ -244,6 +246,37 @@ export function unassignedPerson(position1, position2, mergedPosition) {
   }
 }
 
+export function mergedPersonIsValid(mergedPerson) {
+  let msg
+  if (!mergedPerson.role) {
+    msg = "Merged person must have a role"
+  } else if (!mergedPerson.status) {
+    msg = "Merged person must have a status"
+  } else if (!mergedPerson.rank) {
+    msg = `Merged person must have a ${Settings.fields.person.rank}`
+  } else if (!mergedPerson.gender) {
+    msg = `Merged person must have a ${Settings.fields.person.gender}`
+  } else if (!mergedPerson.country) {
+    msg = `Merged person must have a ${Settings.fields.person.country}`
+  } else if (mergedPerson.role === Person.ROLE.ADVISOR) {
+    if (!mergedPerson.emailAddress) {
+      msg = `${
+        Settings.fields.person.emailAddress.label
+      } is required for ${Person.humanNameOfRole(Person.ROLE.ADVISOR)}`
+    } else if (!mergedPerson.endOfTourDate) {
+      msg = `${
+        Settings.fields.person.endOfTourDate
+      } is required for ${Person.humanNameOfRole(Person.ROLE.ADVISOR)}`
+    }
+  }
```

Comment:
Alternatively you could combine missing requirements in single message, to prevent each error appears as a surprise to the user.

Reply:
Sounds like a good improvement: turn `msg` into an array and show it neatly joined up in the toast message, @anilsonmez-simsoft ?